cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -88,6 +88,18 @@ data:
             proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
             error_page 401 = /login;
         }
+        location /healthz {
+            return 200;
+            add_header Content-Type text/plain;
+        }
+        location /api/healthz {
+            return 200;
+            add_header Content-Type text/plain;
+        }
+        location /model/healthz {
+            return 200;
+            add_header Content-Type text/plain;
+        }
```

Comment:
I don't quite understand the implications of this change.

- Is NGINX serving requests considered suitable for "cost-analyzer-frontend is healthy"?
- Why do we need to do `/api/healthz` and `/model/healthz` on top of `/healthz`? AFAICT K8s only hits `/healthz` [for the frontend readinessProbe](https://github.com/kubecost/cost-analyzer-helm-chart/blob/ef9b541880070ccb2b33171abe3411ed916bb404/cost-analyzer/templates/cost-analyzer-deployment-template.yaml#L694)
- I assume this doesn't break the cost-model readiness probe because its a different container and K8s hits the container directly for healthz?